### PR TITLE
Portuguese Language Updates

### PR DESCRIPTION
**Pull request by Ustklm**
(In progress)

### CODE REVIEW

**Comment by yairm210** [comment]
Please resolve the conflicts =)

**Comment by Ustklm** [comment]
Which conflicts?

**Comment by yairm210** [comment]
image

**Comment by Ustklm** [comment]
I Think that i solved the conflicts now.

File: android/assets/jsons/Translations/Buildings.json [hunk 1]
```diff
@@ -573,7 +586,8 @@
 		German:"Manhattan-Projekt"
 		Polish:"Projekt Manhattan"
                 Malay:"Projek Manhattan"
-		Korean:"맨해튼 프로젝트"
+		Korean:"맨해튼 프로젝트
```
Comment (yairm210) [inline android/assets/jsons/Translations/Buildings.json]: Add a " here, that's probably why the Travis build is failing

**Comment by Ustklm** [comment]
Alright, i added the ", now i am translating some things, and checking if i missed some more ".

**Comment by Ustklm** [comment]
Everything is done on that Portuguese translation update's, so, I am closing that pull.
-Translated everything for portuguese
-Minor gramatical fixes
{ Mini-Updates fixed some conflicts and digitation error }

**Pull request merged**